### PR TITLE
Refactor Login to use Bloc Library

### PR DESCRIPTION
**Pull request by felangel**
- Formatted entire project
- Refactored Login to use the bloc library

### CODE REVIEW

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) please, pay attention to this pull request

**Comment by 0crat** [comment]
Job [#89](https://github.com/ammaratef45/attendance/issues/89) is now in scope, role is `REV`

**Comment by 0crat** [comment]
@ammaratef45[/z](https://www.0crat.com/u/ammaratef45) everybody who has role `REV` is banned at [#89](https://github.com/ammaratef45/attendance/issues/89); I won't be able to assign anyone automatically; consider assigning someone manually (as in [§19](http://www.zerocracy.com/policy.html#19)), or invite more people (as in [§51](http://www.zerocracy.com/policy.html#51)), or remove the job from the scope (as in [§14](http://www.zerocracy.com/policy.html#14))

**Review by ammaratef45 — APPROVED** [review]
@felangel I'm really thankful.
This PR will guide me refactoring the code using your nice library.

**Comment by ammaratef45** [comment]
@rultor merge

**Comment by 0crat** [comment]
Job `gh:ammaratef45/attendance#89` is not assigned, can't get performer

**Comment by 0crat** [comment]
The job [#89](https://github.com/ammaratef45/attendance/issues/89) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @ammaratef45[/z](https://www.0crat.com/u/ammaratef45)

File: attendance/test/api_test.dart [hunk 1]
```diff
@@ -5,34 +5,26 @@ import 'package:attendance/backend/session.dart';
...
       //'python',
-      <String>[
-        'firebase_token_generator.py',
-        'GUfzhtGu1vVFJaYIvxi1yIa49Oy1'
-      ]
-    );
-    final String t = p.stdout.toString().trim();
-    print(p.stderr);
-    return t;
+      <String>['firebase_token_generator.py', 'GUfzhtGu1vVFJaYIvxi1yIa49Oy1']);
```
Comment (webstoreportal) [inline attendance/test/api_test.dart:11]: Cool script run token generator

**Review by webstoreportal — COMMENTED** [review]
Cool test script run token generator

**Pull request merged**